src/routes/note-create.test.js

```diff
@@ -0,0 +1,43 @@
+import { MemoryRouter } from "react-router-dom";
+import { render, cleanup, act } from '@testing-library/react';
+
+import NoteCreateRoute from './note-create';
+import Harness from '../../test/jest/helpers/harness';
+
+const location = {
+  pathname: 'pathname',
+  search: '',
+  hash: '',
+};
+
+const history = {
+  goBack: jest.fn()
```

Comment:
Don't forget about commas after each value in object
```suggestion
  goBack: jest.fn(),
```